pkgs/racket-doc/scribblings/reference/units.scrbl

```diff
@@ -314,7 +319,17 @@ treated as a kind of import into the local definition context. That
 is, for every binding that would be available in a unit that used the
 @racket[export] clause's @racket[tagged-sig-spec] as an import, a
 definition is generated for the context of the
-@racket[define-values/invoke-unit] form.}
+@racket[define-values/invoke-unit] form.
+
+If no @racket[maybe-results-clause] is provided, the unit body may return
+any number of values, all of which are ignored. Otherwise, the values
+returned from the unit body are bound to the given @racket[result-id]s,
+in order. If no @racket[rest-results-id] is provided, the body must return
+exactly as many values as there are @racket[result-id]s, but if it is
+provided, the body may return arbitrarily many more, and
+@racket[rest-result-id] is bound to a list containing the extra results.
+
+@history[#:changed "8.8.0.2" @elem{Added @racket[maybe-results-clause].}]}
```

Comment:
We're up to 8.8.0.6. Bumping to 8.8.0.7 (by changing `racket/src/version/racket_version.h` and `pkg/base/info.rkt`) would make sense, especially in combination with the TR change.